Assets/General/Audio/FMODScripts/IntervalAmbienceEvents.cs

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+// File Name:       IntervalAmbienceEvents.cs
+// Author:          Mark Hanson
+// Creation Date:   11/14/2024
+//
+// Description:     Handler of random ambient sound effects at random times
+******************************************************************************/
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using FMODUnity;
+using Random = UnityEngine.Random;
+/// <summary>
+/// Handler of random ambient sound effects at random times
+/// </summary>
+public class IntervalAmbienceEvents : MonoBehaviour
+{
+  /// <summary>
+  /// Struct to hold min and max time for random range
+  /// </summary>
+    public struct FModEventReference
+    {
+        public int MinTimeBetweenEvents;
+        public int MaxTimeBetweenEvents;
+    }
+
+    public FModEventReference FmodEvents { get; set; }
+   [SerializeField] private EventReference[] _ambienceArray;
+
+   private Coroutine _ambienceCoroutine;
+   /// <summary>
+   /// Start coroutine 
+   /// </summary>
+   private void Start()
+   {
+       if (_ambienceCoroutine == null)
+       {
+           StartCoroutine(PlayIntervalAudio());  
```

Comment:
You have the _ambienceCoroutine variable but it is never set. This line should be 
_ambienceCoroutine = StartCoroutine(PlayIntervalAudio());